training-maquetado/scss/list.scss

```diff
@@ -1,8 +1,8 @@
 .book-list {
+  background-color: $wildsand;
   display: flex;
   flex-wrap: wrap;
-  max-width: 1366px;
-  width: 100%;
+  padding: $navbar-height + 36px 234px;
```

Comment:
I'm not sure why is this working or if this is correct

Reply:
mmm this is weird. why do you need this?